hs_core/views/discovery_json_view.py

```diff
@@ -0,0 +1,59 @@
+import simplejson as json
```

Comment:
@ironfroggy It seems simplejson updates more frequently than built-in json library, so that we can use the latest version of json in python by importing simplejson. Json and simplejson both has pros and cons, as far as I know, built-in json library has a faster dumping speed and simplejson is faster on loading and is updated more frequently. If you think it is better to use the built-in json library, I switch to it.